package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "iobroker.smartmeter",
-  "version": "3.2.0",
+  "version": "3.2.1-0",
   "description": "Smartmeter Adapter",
   "author": "Ingo Fischer <ingo@fischer-ka.de>",
-  "contributors": [],
+  "contributors": [
+  @DutchmanNL],
```

Comment:
This is invalid or?!

Reply:
why  ? using the @ creates a hyperlink in GitHub directing to the user page

Reply:
Because it is in a json file and this is no proper JSON! and fromm my knowledge not siupported by package.json

Reply:
solved